x-pack/plugins/security_solution/public/common/components/header_section/__snapshots__/index.test.tsx.snap

```diff
@@ -6,6 +6,7 @@ exports[`HeaderSection it renders 1`] = `
 >
   <EuiFlexGroup
     alignItems="center"
+    direction="row"
```

Comment:
Can we add the explanation for this change as a response to that comment?

Reply:
Which comment?

The default value of direction is `row`. So it doesn't change the UI. 
We had to update the snapshot file because now we are always setting the `direction` property. 

File:  x-pack/plugins/security_solution/public/common/components/header_section/index.tsx
Line: 77

https://github.com/elastic/kibana/pull/122980/files#diff-6226985ec8ed95a05bda7b3eff1a40abaf99c311b0109a02272d7d03911c7609R77